LICENSE

```diff
@@ -1,21 +1,26 @@
-MIT License
+BSD 2-Clause License
```

Comment:
The official name is `2-Clause BSD`, isn’t it?

Reply:
No. "BSD 2-Clause" when space is acceptable else "BSD-2-Clause".
[default GitHub template](https://github.com/MediaArea/RAWcooked/community/license/new?branch=master&template=bsd-2-clause)
[SPDX entry for BSD 2-Clause](https://spdx.org/licenses/BSD-2-Clause.html)

Reply:
Hmm… on their own [website](https://opensource.org/licenses/BSD-2-Clause) it’s some ambiguous: 
<img width="480" alt="bildschirmfoto 2018-03-23 um 18 59 17" src="https://user-images.githubusercontent.com/3246607/37845965-6f563fac-2ecc-11e8-8470-080bb93cff3d.png">

Reply:
This is not the BSD website.
[FreeBSD license](https://www.freebsd.org/copyright/freebsd-license.html)

I strongly suggest to keep GitHub template so it is correctly flagged by GitHub, and SPDX (standard for license strings in open source repositories) has the same naming order too.

Reply:
It’s fine with me.